### DISCUSSION TITLE
how to create a Structured_Build_Layout  with bundles\features\tests\sites\products

### DISCUSSION THREAD

**Discussion by w6et** [eclipse-tycho/tycho#2837] [Q&A]
step1: configuration settings.xml 
        reference：https://github.com/eclipse-tycho/tycho/wiki
setp2:
        reference：https://tycho.eclipseprojects.io/doc/latest/StructuredBuild.html
then build with 'mvn clean verify',Some problems were encountered cause by 'bundles\features\sites\products module not exist '
then comment 4 modules,add a test moudle
       reference：https://tycho.eclipseprojects.io/doc/latest/TestingBundles.html
`	<packaging>eclipse-test-plugin</packaging>

	<build>
		<plugins>
			<plugin>
				<groupId>org.eclipse.tycho</groupId>
				<artifactId>tycho-surefire-plugin</artifactId>
				<version>${tycho.version}</version>
			<executions>
				<execution>
					<id>execute-integration-tests</id>
					<goals>
						<goal>bnd-test-test</goal>
						<goal>verify</goal>
					</goals>
				</execution>
			</executions>
			</plugin>
		</plugins>
	</build>`
then build with 'mvn clean verify' or 'mvn tycho-surefire-plugin:plugin-test',encountered '\META-INF\MANIFEST.MF: Manifest file not found'

I wanted to ask for help how to build a complete Structured_Build_Layout  with bundles\features\tests\sites\products

**Comment by laeubi** [comment]
@w6et it is quite unclear what you try to archive here and what errors you see, I think the best would be to share a little example that shows the error
